mgr/RootCint.pl

```diff
@@ -437,9 +440,10 @@
     #	$cmd  = "rootcint -f $Cint_cxx -c  -D__NO_STRANGE_MUDST__ -DROOT_CINT -D__ROOT__ $CPPFLAGS $h_files";
     #	print "cmd (+extra) = ",$cmd,"\n";
     #} else {
-    	$cmd  = "rootcint -f $Cint_cxx -c -DROOT_CINT -D__ROOT__ $CPPFLAGS $h_files";
+##    	$cmd  = "rootcint  -f $Cint_cxx -c -DROOT_CINT -D__ROOT__ $CPPFLAGS $h_files";
+    	$cmd  = "rootcling -rmf $Cint_map -f $Cint_cxx  -DROOT_CINT -D__ROOT__ $CPPFLAGS $h_files";
```

Comment:
Does this work for our ROOT5 builds as well?  (This commit _cannot_ break the ROOT5 build.)

Reply:
... probably should have logic testing against the root version we compile against, and use root5/rootcint versus root6/rootcling.